modules/user-data/templates/entrypoint.sh.tpl

```diff
@@ -482,7 +481,31 @@ else
   fi
 fi
 
+echo "#########################"
+echo "#    Setup Telegraf     #"
+echo "#########################"
+
+# Installs Telegraf
+curl -s https://repos.influxdata.com/influxdata-archive.key > influxdata-archive.key
```

Comment:
Move the installation in the Packer scripts?

Reply:
I was wondering whether it should be there or not, still not entirely positive that we want Telegraf to be part of the VM image

Reply:
Then at least move the GraphDB start after configuring the monitoring in order to to capture metrics from the beggingin.

Reply:
> still not entirely positive that we want Telegraf to be part of the VM image

I don't see why not. Having everything needed (libs, tools) to run it bundled in the image would greatly reduce the setup and starting time when used.

Reply:
I'm not sure either, let's keep it here for now.

Reply:
Moved it before starting GraphDB